src/components/CommentSummary/index.spec.tsx

```diff
@@ -0,0 +1,183 @@
+import dedent from 'dedent';
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  ExternalComment,
+  createInternalComment,
+} from '../../reducers/comments';
+import { createFakeExternalComment, fakeVersion } from '../../test-helpers';
+
+import CommentSummary, { PublicProps } from '.';
+
+describe(__filename, () => {
+  const createComments = (comments: Partial<ExternalComment>[]) => {
+    return comments.map((c) => {
+      return createInternalComment(createFakeExternalComment(c));
+    });
+  };
+
+  const render = ({
+    comments = createComments([createFakeExternalComment()]),
+    ...props
+  }: Partial<PublicProps> = {}) => {
+    return shallow(<CommentSummary comments={comments} {...props} />);
+  };
+
+  const renderComments = (comments: Partial<ExternalComment>[]) => {
+    const root = render({ comments: createComments(comments) });
+    return String(root.find('textarea').prop('value')).trim();
+  };
+
+  const createVersionComment = (props = {}) => {
+    return {
+      filename: null,
+      lineno: null,
+      version: fakeVersion,
+      ...props,
+    };
+  };
+
+  it('accepts a custom className', () => {
+    const className = 'CustomClass';
+
+    expect(render({ className })).toHaveClassName(className);
+  });
+
+  it('renders overall comments', () => {
+    const comment1 = 'First comment';
+    const comment2 = 'Second comment';
+
+    const comments = [
+      createVersionComment({ comment: comment1 }),
+      createVersionComment({ comment: comment2 }),
+    ];
+
+    expect(renderComments(comments)).toEqual(
+      dedent(`
+        Overall comment(s):
+
+        ${comment1}
+
+        ${comment2}
+      `),
+    );
+  });
+
+  it('renders overall comments mixed with file comments', () => {
```

Comment:
Similar to what I mentioned for the Story, there doesn't seem to be a test which includes overall comments, file comments, and file line comments. I think one of those might be worthwhile.